email/1.3/modules/ROOT/pages/email-send.adoc

```diff
@@ -1,136 +1,359 @@
-= Sending Emails with the Email Connector
+= Send Emails with the Email Connector Example - Mule 4
 :keywords: email, connector, configuration, smtp, send, smtps
 :page-aliases: connectors::email/email-send.adoc
 
-Email Connector can send messages over SMTP and SMTPS.
+Anypoint Connector for Email (Email Connector) can send messages over SMTP and SMTPS servers using the *Send* operation. The following examples show how to configure Email Connector to send emails and attachments, both in Anypoint Studio and the XML editor:
 
-== Configuring the Connector
+* <<send-email,Send Emails>> +
+Configure the Email Connector *Send* operation to send emails over a SMTPS server.
 
-Separate configuration types are available for listing emails stored in a server:
+* <<send-attachments,Send an Attachment>> +
+Configure the File Connector *Read* operation to read a JSON file, and the Email Connector *Send* operation to send the file as an attachment over a SMTP server.
 
-* SMTP
-* SMTPS (secured by TLS)
+* <<send-attachments,Send Multiple Attachments>> +
+Configure *Set Variable* components to define multiple media type attachments, and send these attachments using the Email Connector *Send* operation. Subsequently, log the result message using a *Logger* component.
```

Comment:
Global: "the Email Connector" > "Email Connector" (I prefer the way you have it, but Kim feels strongly about not using "the" before the connector name)